HIP/hip-655.md

```diff
@@ -0,0 +1,106 @@
+---
+hip: 655
+title: Token airdrops without explicit associations
+author: Nick Poorman <@nickpoorman>
+working-group: Richard Bair <@rbair23>, Jasper Potts <@jasperpotts>, Atul Mahamuni <@atul-hedera>, Michael Tinker <@tinker-michaelj>, Leemon Baird <@lbaird>
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Review
+created: 2022-10-20
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/654
+updated: 2023-01-20, 2023-01-24, 2023-02-14
+---
+
+## Abstract
+
+[HIP-23](https://hips.hedera.com/hip/hip-23) allowed an account owner to open up a specified number of automatic token-association slots into which others can transfer tokens. However, this capability doesn't allow for a pure "airdrop" of tokens. This HIP provides an additional mechanism for senders to airdrop tokens to other accounts, while allowing individual account holders to maintain control over what shows up in their account balances.
```

Comment:
suggestion: to separate concept of balance with association. Depending on the implementation a group of associated tokens and those in the inbox could be grouped in a single list for UX.
Though uncertain it's fair to be specific that the user controls the association of their account
```suggestion
[HIP-23](https://hips.hedera.com/hip/hip-23) allowed an account owner to open up a specified number of automatic token-association slots into which others can transfer tokens. However, this capability doesn't allow for a pure "airdrop" of tokens. This HIP provides an additional mechanism for senders to airdrop tokens to other accounts, while allowing individual account holders to maintain control over what token they are associated with.
```